SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -831,10 +848,15 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
         pw = max(pw, sust_w);
       }
 
+      if(sstParsedOptions.production == SST_OPTIONS::COMP_Sarkar) {
+        Dilatation_Sarkar = -0.15 * pk * Mt + 0.2 * beta_star * (1.0 +zetaFMt) * Density_i * ScalarVar_i[1] + ScalarVar_i[0] * Mt * Mt;
+        pk += Dilatation_Sarkar;
```

Comment:
Can you check the equation for the pressure dilatation that you give above and in your PR description? Should I compare it to eq (39) of the paper? The equation (39) in the paper is (-a2*P*Mt^2) +(a3*rho*eps*Mt^2) which is different from what you write above and what is in the code above. I guess you use the relationship eps=beta^{*}*omega*k, so the expression from Rumsey becomes:
(-a2*P*Mt^2) +(a3*beta^{*}*rho*omega*k*Mt^2)

You have in the code (-a2*P*Mt) + (a3*beta^{*}*rho*omega) + (k*Mt^2)

So it seems as if the "+" should be a multiplication? It also looks like there is an Mt term missing from the first term

Reply:
Thank you for your thoroughly code review. 

Yes, you're right. It's definitely my mistake. I will fix the code and regression test case.